### DISCUSSION TITLE
RandomCrop with padding

### DISCUSSION THREAD

**Discussion by hassiahk** [albumentations-team/albumentations#940] [Q&A]
I want to implement an equivalent of Torchvision's transforms (`transforms.RandomCrop(32, padding=4)`) in albumentations.

The only way I found till now is:

```
A.Sequential([
    A.PadIfNeeded(
        min_height=32+8,
        min_width=32+8,
    ),
    A.RandomCrop(
        height=32,
        width=32
    )
], p=1.0)
```

What are the different ways we can do this? And what is an efficient or recommended way?

Thanks in advance!! :)

**Comment by BloodAxe** [comment]
Your snippet looks very legit to me. You can also play with A.RandomSizedCrop to introduce some scale variance to your crops.
